src/features/rewards/walletPanel/index.tsx

```diff
@@ -90,25 +91,27 @@ export default class WalletPanel extends React.PureComponent<Props, {}> {
 
     return (
       <StyledSelectWrapper>
-        <Select
-          floating={true}
-          showAllContents={true}
+        <StyledSelect
           value={monthlyAmount}
           onChange={this.props.onAmountChange}
         >
           {donationAmounts.map((token: Token, index: number) => {
+            const tokenValue = token.tokens.toString()
+            const paddingLength = tokenValue.length < 5
+              ? tokenValue.length === 4 ? 3 : 4
```

Comment:
The number of space entities that look best for values of length 4 and 3 are 3 and 4 respectively. It didn't end up just being some math on the length - 5.